azerothcore/azerothcore-wotlk
# fix(DB/Quest): The Key of Warlord Zol'Maz

Fix quest The Key of Warlord Zol'Maz

##### CHANGES PROPOSED:

- Fix quest The Key of Warlord Zol'Maz where gameobject ''Gate'' didn't activate
- Updated SAI for Warlord Zol'Maz


###### ISSUES ADDRESSED:


Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1852


##### TESTS PERFORMED:

Yes, tested by @Knindzagxg and me. Everything fine ^^


##### HOW TO TEST THE CHANGES:

1. .go c 115833
2. quest add .quest add 12712
3. add 39314
4. Use Tiki Dervish Ceremony and summon Guardian / Doors should be opened after speak of NPC, and he will attack guardian&player


##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master
